readme.txt

```diff
@@ -16,7 +16,7 @@ Twenty Twenty-Four is designed to be flexible, versatile and applicable to any w
 = 1.0 =
 * Released: November 7, 2023
 
-https://wordpress.org/support/article/twenty-twenty-four-changelog#Version_1.0
+https://wordpress.org/documentation/article/twenty-twenty-four-changelog#Version_1.0
```

Comment:
```suggestion
https://wordpress.org/documentation/article/twenty-twenty-four-changelog/#Version_1.0
```

The article would redirect to a URL with the slash (once it is published). I also [suggested the possibility](https://core.trac.wordpress.org/ticket/58832#comment:14) of "Initial release" instead of a link for this version (so it would not require immediately publishing the article).

Reply:
I think it's fine if it's not an immediate thing, it's not going to be long for the link to work